app/controllers/health/signable_documents_controller.rb

```diff
@@ -5,8 +5,140 @@
 ###
 
 module Health
-  class SignableDocumentsController < Window::Health::SignableDocumentsController
+  class SignableDocumentsController < IndividualPatientController
     include ClientPathGenerator
+    include HealthCareplan
+
+    helper ChaHelper
+
+    before_action :set_client, except: [:signature, :signed]
+    before_action :set_patient, except: [:signature, :signed]
+    before_action :set_careplan, except: [:signature, :signed]
+    before_action :set_medications, except: [:signature, :signed]
+    before_action :set_problems, except: [:signature, :signed]
+
+    # This supports signing without logging in:
+    skip_before_action :authenticate_user!, only: [:signature, :signed]
+    skip_before_action :require_some_patient_access!, only: [:signature, :signed]
+
+    def create
+      @team = @careplan.team
+
+      @signers = []
+      @signers << { 'email': @patient.current_email, 'name': @patient.name }
+
+      @doc = @careplan.signable_documents.build(signers: @signers, primary: true, user_id: current_user.id)
+
+      @doc.pdf_content_to_upload = get_pdf
+
+      if @doc.valid?
+        @careplan.class.transaction do
+          @doc.save
+          @careplan.signable_documents.where.not(id: @doc.id).update_all(primary: false)
+          @expires_at = Time.now + signature_source.expires_in
+          @signature_request = signature_source.create!(
+            patient_id: @patient.id,
+            careplan_id: @careplan.id,
+            to_email: @patient.current_email,
+            to_name: "#{@patient.first_name} #{@patient.last_name}",
+            requestor_email: current_user.email,
+            requestor_name: current_user.name,
+            sent_at: Time.now,
+            expires_at: @expires_at,
+            signable_document_id: @doc.id
+          )
+          @doc.make_document_signable!
+        end
+
+        flash[:notice] = "Careplan signature requested from #{@doc.signers.map(&:email).join('; ')}"
+      else
+        flash[:error] = "#{@doc.errors.full_messages.join('. ')}"
+      end
+      url_params = {client_id: @client.id, careplan_id: @careplan.id, id: @doc.id, email: @patient.current_email, hash: @doc.signer_hash(@patient.current_email)}
+      url_params[:sign_out] = true if params[:sign_out].present?
+      redirect_to polymorphic_path([:signature] + careplan_path_generator + [:signable_document], url_params)
+      # redirect_back fallback_location: client_health_careplans_path(@client)
+    end
+
+    # def remind
```

Comment:
Remove?

Reply:
I think there's still a possibility of needing this in the future and it's nice documentation of how.